### PR TITLE
CHANGELOG for 1.4.0

### PR DESCRIPTION
**Pull request by xorye**
Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+ * Show artifactId in the extensions picker. See [#197](https://github.com/redhat-developer/vscode-quarkus/issues/197)
+ * Add support for MicroProfile Fault Tolerance. See [#174](https://github.com/redhat-developer/vscode-quarkus/issues/174)
+ * Manage client snippet on server side. See [#119](https://github.com/redhat-developer/vscode-quarkus/issues/119)
+ * Determine if current project is a Quarkus project before adding Quarkus extensions.. See [#45](https://github.com/redhat-developer/vscode-quarkus/issues/45)
+ * Two outputs in the "Tasks" dropdown. See [#17](https://github.com/redhat-developer/vscode-quarkus/issues/17)
+
+### Bug Fixes
+
+ * Bad perfomance when working with non quarkus/MP projects . See [#240](https://github.com/redhat-developer/vscode-quarkus/issues/240)
+ * Fix for vulnerable minimist dependency. See [#238](https://github.com/redhat-developer/vscode-quarkus/pull/238)
```
Comment (fbricon) [inline CHANGELOG.md]: build

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+ * Excluding unknown properties from validation no longer work. See [#237](https://github.com/redhat-developer/vscode-quarkus/issues/237)
+ * Improve application.properties textmate grammar. See [#233](https://github.com/redhat-developer/vscode-quarkus/issues/233)
+ * Update application.properties TextMate grammar for multi-line properties values. See [#229](https://github.com/redhat-developer/vscode-quarkus/issues/229)
```
Comment (fbricon) [inline CHANGELOG.md:32]: use consistent casing for TextMate, see line above

**Review by fbricon — CHANGES_REQUESTED** [review]
this changelog should also list the visible enhancements and fixes from https://github.com/redhat-developer/quarkus-ls/pull/305/

The server side urls should look like:
`Support for `MicroProfile LRA` properties. See [LS#243](https://github.com/redhat-developer/quarkus-ls/issues/243)`

**Comment by xorye** [comment]
updated

**Comment by xorye** [comment]
updated to add the 2 new issues from quarkus-ls

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+ * Improve application.properties TextMate grammar. See [#233](https://github.com/redhat-developer/vscode-quarkus/issues/233)
+ * Update application.properties TextMate grammar for multi-line properties values. See [#229](https://github.com/redhat-developer/vscode-quarkus/issues/229)
+ * Qute Syntax coloration problem with quote. See [#221](https://github.com/redhat-developer/vscode-quarkus/issues/221)
+ * When pasting, the comments gets duplicated when "editor.formatOnPaste":true. See [#220](https://github.com/redhat-developer/vscode-quarkus/issues/220)
+ * Cannot select which project to debug when multiple workspace folders are open. See [#74](https://github.com/redhat-developer/vscode-quarkus/issues/74)
+ * Remove duplicate Quarkus outputs in the "Tasks" dropdown. See [#17](https://github.com/redhat-developer/vscode-quarkus/issues/17)
+ * Duplicate static properties after saving Java files. See [LS#301](https://github.com/redhat-developer/quarkus-ls/issues/301)
+ * Bad performance when working with non Quarkus/MP projects. See [LS#290](https://github.com/redhat-developer/quarkus-ls/pull/290)
```
Comment (fbricon) [inline CHANGELOG.md]: duplicates #240

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+ * Allow excluded unknown property pattern * to match forward slashes. See [LS#284](https://github.com/redhat-developer/quarkus-ls/pull/284)
```
Comment (fbricon) [inline CHANGELOG.md]: duplicates #237

**Review by fbricon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (xorye) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 4 above)
Comment (xorye) [reply CHANGELOG.md]: removed

**Comment by fbricon** [comment]
Thanks @xorye!

**Pull request merged**